Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -124,16 +124,26 @@ struct KalmanFitterResult {
   MultiTrajectory<source_link_t> fittedStates;
 
   // This is the index of the 'tip' of the track stored in multitrajectory.
+  // This correspond to the last measurment state in the multitrajectory.
   // Since this KF only stores one trajectory, it is unambiguous.
   // SIZE_MAX is the start of a trajectory.
   size_t trackTip = SIZE_MAX;
 
+  // This is the index of the 'tip' of the states stored in multitrajectory.
+  // This correspond to the last state in the multitrajectory.
+  // multitrajectory. Since this KF only stores one trajectory, it is
+  // unambiguous. SIZE_MAX is the start of a trajectory.
+  size_t endTip = SIZE_MAX;
+
   // The optional Parameters at the provided surface
   std::optional<BoundTrackParameters> fittedParameters;
 
   // Counter for states with measurements
   size_t measurementStates = 0;
 
+  // Counter for measurements holes
```

Comment:
I think it might be good to add some explanation here how we count the holes.

Reply:
Great idea !